narwhal/node/tests/staged/narwhal.yaml

```diff
@@ -85,10 +85,7 @@ WorkerIndex:
         TYPENAME: WorkerInfo
 WorkerInfo:
   STRUCT:
-    - name:
-        TUPLEARRAY:
-          CONTENT: U8
-          SIZE: 32
+    - name: STR
```

Comment:
I think this is expected, the underlying structure for the `NetworkPublicKey`, which is the `name` field of the `WorkerInfo` struct changed. @joske might have more insights to provide.

Reply:
yes this was already in the original PR